### PR TITLE
Add dataservice show page

### PR DESCRIPTION
**Pull request by nicolaskempf57**
Requires https://github.com/opendatateam/udata/pull/3049
Closes https://github.com/datagouv/data.gouv.fr/issues/1369

### CODE REVIEW

**Review by ThibaudDauce — APPROVED** [review]
I quickly checked the page, it seems really nice! I don't know if we want to merge it as it and iterate to avoid big PR waiting for a long time?

Just a question about all the `ugc` / `noopener`, could we extract this outside this PR? It doesn't seems related…

**Comment by nicolaskempf57** [comment]
> I quickly checked the page, it seems really nice! I don't know if we want to merge it as it and iterate to avoid big PR waiting for a long time?

Indeed, the goal is to have a first version available for the next release and iterate on it later.

> Just a question about all the `ugc` / `noopener`, could we extract this outside this PR? It doesn't seems related…

I changed it for the dataservice`<a>` so I found it odd to keep the less secure ones unchanged. But if you prefer I can extract them to a new PR.

**Comment by ThibaudDauce** [comment]
> > I quickly checked the page, it seems really nice! I don't know if we want to merge it as it and iterate to avoid big PR waiting for a long time?
> 
> Indeed, the goal is to have a first version available for the next release and iterate on it later.
> 
> > Just a question about all the `ugc` / `noopener`, could we extract this outside this PR? It doesn't seems related…
> 
> I changed it for the dataservice`<a>` so I found it odd to keep the less secure ones unchanged. But if you prefer I can extract them to a new PR.

Good for me!

File: udata_front/views/dataservice.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+            author_name = dataservice.organization.name
+            author_uri = url_for('organizations.show',
+                                 org=dataservice.organization.id, _external=True)
+        elif dataservice.owner:
+            author_name = dataservice.owner.fullname
+            author_uri = url_for('users.show',
+                                 user=dataservice.owner.id, _external=True)
+        feed.add_item(dataservice.title,
+                      description=dataservice.description,
+                      content=render_template('dataservice/feed_item.html',
```
Comment (maudetes) [inline udata_front/views/dataservice.py:35]: This feed does not exist

File: udata_front/views/dataservice.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        if not DataserviceEditPermission(self.dataservice).can():
+            if self.dataservice.private:
+                abort(404)
+            elif self.dataservice.deleted:
+                abort(410)
+        context['can_edit'] = DataserviceEditPermission(self.dataservice)
+        return context
+
+
+@sitemap.register_generator
```
Comment (maudetes) [inline udata_front/views/dataservice.py:78]: :+1:

**Review by maudetes — COMMENTED** [review]
Wow! Nice already!

Indeed, we can probably iterate on this base later on :)

File: udata_front/views/dataservice.py (hunk 1 above)
Comment (nicolaskempf57) [reply udata_front/views/dataservice.py:35]: Added and fixed !

**Review by maudetes — APPROVED** [review]
:ok_hand:

**Pull request merged**